distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/search-ui.json

```diff
@@ -1,10 +1,8 @@
 {
   "name": "search-ui-app",
-  "description": "The Search UI is an application that not only provides results in a html format but also provides a convenient, simple querying user interface.\nThe left pane of the SearchUI contains basic fields to query the Catalog and other Sources. The right pane consists of a map.\nIt includes both standard (3d globe) and simple (text page) versions.::Search UI",
+  "description": "The Search UI is an application that not only provides results in a html format but also provides a convenient, simple querying user interface.::Search UI",
```

Comment:
📉 Note: Other projects that install another search UI will need to
- modify this file for the configurations to show up under the `Search UI` app in the old Admin Console.
- update the `Redirect URI` configuration to point to the new uri